cli/cmd/install.go

```diff
@@ -8,7 +8,7 @@ import (
 	"text/template"
 
 	"github.com/runconduit/conduit/controller"
-	uuid "github.com/satori/go.uuid"
+	"github.com/satori/go.uuid"
```

Comment:
I don't think this is documented anywhere, but my preference is to explicitly set the package name when the last segment of the import path doesn't match the name of the package being imported.